packages/mendel-exec/index.js

```diff
@@ -136,6 +136,11 @@ module.exports = {
 
                     let normId = parent.deps[depLiteral][runtime];
                     if (typeof normId === 'object') normId = normId[runtime];
+
+                    // If we get _noop from cache, this depLiteral doesn't exist
+                    if (normId === '_noop')
+                        throw new Error(`Cannot find ${depLiteral} from ${mainEntry.id}`); // eslint-disable-line max-len
+
```

Comment:
@irae After this error, we have a [catch block](https://github.com/yahoo/mendel/pull/108/files/1c12bb9cf3015c9a62920a7a7f8498ab302df8f4#diff-74a2a21565400d96ccd47fe47532fadeR147) that is logging these errors using `console.error`. We also have another scenario in same code block where we [throw RangeError](https://github.com/yahoo/mendel/pull/108/files/1c12bb9cf3015c9a62920a7a7f8498ab302df8f4#diff-74a2a21565400d96ccd47fe47532fadeL135).

Since we have a catch, process doesn't exit for this particular error. It just logs them on console. On updating file, client resumes sync and other operations.

```
GET / 500 1098 - 25.549 ms
Error: Cannot find ./button1 from ./isomorphic/base/main.js
    at resolver (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:142:31)
    at varRequire (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:93:23)
    at ./isomorphic/base/components/body.js:25:16
    at runEntryInVM (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:30:9)
    at varRequire (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:95:20)
    at ./isomorphic/base/components/app.js:17:13
    at runEntryInVM (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:30:9)
    at varRequire (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:95:20)
    at ./isomorphic/base/main.js:3:12
    at runEntryInVM (/Users/muralikr/dev/workspace/mendel/packages/mendel-exec/index.js:30:9)
[Mendel] File change detected. Waiting to sync again...
[Mendel] Synced
```

I felt that current code is better since it shows proper error message with call stack. Please let me know if you want to change this to `console.error`. I am ok with either ways.